src/Vue/README.md

```diff
@@ -0,0 +1,14 @@
+# Symfony UX Vue.js
+
+Symfony UX Vue integrates [Vue.js](https://vuejs.org/) into Symfony applications.
+It provides tools to render Vue.js components from Twig.
```

Comment:
We can't do it until after the PR is merged (and the subtree split is setup), but we will also need to add a demo to ux.symfony.com and linking to it from here would be greta.

Reply:
Passing by and reminded me one of my PR https://github.com/symfony/symfony-docs/pull/16403
 Could it be great to show case example of vuejs mounting with object serialized given to?
More a real ex as just passing a « name »

Reply:
I'm already planning to add a ux.symfony.com example (maybe in a following PR ?).

I'll link it here when ready :+1:

Reply:
Great 👍🏻